superset/connectors/sqla/models.py

```diff
@@ -1438,15 +1438,24 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                 select_exprs.append(outer)
         elif columns:
             for selected in columns:
+                if is_adhoc_column(selected):
+                    sql = selected.get("sqlExpression") or ""
```

Comment:
@zhaoyongjie now as `sqlExpression` in `AdhocColumn` is optional, I need to put `or ""` to tell mypy it's a string, is this correct way?

Reply:
can we directly do selected["sqlExpression"] as we already checked that in is_adhoc_column?
so we get definite str type on _sql and _column_label.

Reply:
sure. the`.get` and square brackets are the same meaning in this case.